components/Header.vue

```diff
@@ -71,14 +71,14 @@ export default {
   }
 
   .cartcount {
-    font-family: 'Barlow', sans-serif;
+    font-family: "Barlow", sans-serif;
     position: absolute;
     background: #ff2211;
     color: white;
     text-align: center;
     padding-top: 2px;
     height: 18px;
-    width: @height;
+    width: 18px;
```

Comment:
Possibly this could be created as an scss variable and just referenced two times, but for such a simple example it's alright. Sass doesn't support referencing other properties, right?

Reply:
Good idea, I'll update that